app/views/events/index.html.erb

```diff
@@ -20,7 +20,9 @@
     <% end %>
     <% content_for :display_options do %>
       <ul class="nav nav-xs nav-pills index-display-options">
+        <%# We should consider setting the active status based on the query fragment so we can deeplink %>
```

Comment:
I was under the impression that the tab history JavaScript did this dynamically after the page loads